app/definitions/ISubscription.ts

```diff
@@ -18,6 +18,14 @@ export enum SubscriptionType {
 	E2E = 'e2e', // FIXME: this is not a type of subscription
 	THREAD = 'thread' // FIXME: this is not a type of subscription
 }
+export enum ReverseSubscriptionType {
+	'p' = SubscriptionType.GROUP,
+	'd' = SubscriptionType.DIRECT,
+	'c' = SubscriptionType.CHANNEL,
+	'l' = SubscriptionType.OMNICHANNEL,
+	'e2e' = SubscriptionType.E2E,
+	't' = SubscriptionType.THREAD
+}
```

Comment:
I think later in the comment you have mentioned the usage of roomTypeToApiType, but here we need an enumeration that maps the string literals 'p', 'd', etc, to Subscription type of the room, since the function getRoomMembers requires it in this data type only.

Reply:
An alternative to using this could be add an async call to getRoom function from lib/methods/getRoom and then get the roomSubscriptionType from there.
Right now I am receiving the roomType and rid as a string through props.